app/views/profiles/search.html.erb

```diff
@@ -0,0 +1,98 @@
+<div class = "search-page" id="friend-lookup">
+  
+  <div class = "col-md-6 col-md-offset-3">
+  	<div class="table-responsive">
+		  <table class="table search-table" >
+		    <thead>
+		    	<tr>
+		    		<th class="header search-header">Summoner Search Result</th>
+		    	</tr>
+		    </thead>
+		    <tbody>
+		      <tr>
+		    			<td>
+		    			  <% if @friend %>
```

Comment:
This variable name should be changed.  You can still view the data even if the person is not your friend on the app.  This makes me it seem like the data is only displayed if they are friends with you.

The only thing that should be different depending on thesearched person's friend status with you, is the 'Invite Friend' button at the bottom.  I'm not sure if you have the logic or not, but that button should only be displayed if you are currently logged in and also not already friends with the person in the search.